srcpkgs/tlsdate/template

```diff
@@ -0,0 +1,9 @@
+# Template file for 'tlsdate'
+pkgname=tlsdate
+version=0.0.13
+revision=16
+build_style=meta
```

Comment:
removed-packages should uninstall it and allow people to upgrade their systems, we can also remove it from binary repositories to prevent them from installing it again instead of  adding a metapackage

Reply:
tlsdate<=0.0.13_16 is already in removed-packages so I didn't update it.